runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -31,6 +31,8 @@
 public interface FlinkPipelineOptions
     extends PipelineOptions, ApplicationNameOptions, StreamingOptions {
 
+  String AUTO = "[auto]";
```

Comment:
Should we also add the other options here? Or make this an enum?

Reply:
Did not check if/how enum is supported, but this constant is now only used for the master URL. I think it might be good to get rid of the `[auto]` value for that option as well and just use `null` instead. As separate change though.